planner/core/initialize.go

```diff
@@ -51,6 +51,12 @@ func (ts TableScan) Init(ctx sessionctx.Context, offset int) *TableScan {
 	return &ts
 }
 
+// Init initializes IndexScan.
+func (is IndexScan) Init(ctx sessionctx.Context, offset int) *IndexScan {
```

Comment:
Add prefix `Logical`?

Reply:
And also change `TableScan` to `LogicalTableScan`?